File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    answer = []
    for v in quiz:
        cal,value = v.split(" = ")
        X,op,Y = cal.split()
        result =  0
        if op == "+": result = int(X) + int(Y)
        else: result = int(X) - int(Y)
        answer.append("O") if result == int(value) else answer.append("X")
    return answer

File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.js
function solution(quiz) {
    let answer = [];
    for (let v of quiz){
        let result;
        let [X,op,Y,S,Z] = v.split(" ");
        op === "+"? result = +X + +Y : result = +X - +Y
        result === +Z? answer.push("O") : answer.push("X")
    }
    return answer
}